ci/run_python_tests.sh

```diff
@@ -0,0 +1,171 @@
+#!/bin/bash
+
+set -euo pipefail
+
+test_cmd="${1:-}"
+test_args="${2:-}"
+node_version="${3:-}"
+
+[ "$#" -ne 3 ] && { echo -e "Error: function requires exactly three arguments.\n\n"; exit 1;}
+
+# Functions
+function fn_die() {
+  echo -e "$1" >&2
+  exit "${2:-1}"
+}
+
+function import_gpg_keys() {
+  # shellcheck disable=SC2145
+  printf "%s\n" "Tagged build, fetching keys:" "${@}" ""
+  # shellcheck disable=SC2207
+  declare -r my_arr=( $(echo "${@}" | tr " " "\n") )
+
+  for key in "${my_arr[@]}"; do
+    echo "Importing key: ${key}"
+    gpg -v --batch --keyserver hkps://keys.openpgp.org --recv-keys "${key}" ||
+    gpg -v --batch --keyserver hkp://keyserver.ubuntu.com --recv-keys "${key}" ||
+    gpg -v --batch --keyserver hkp://pgp.mit.edu:80 --recv-keys "${key}" ||
+    gpg -v --batch --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys "${key}"
+  done
+}
+
+function check_signed_tag() {
+  # Checking if git tag signed by the maintainers
+  if git verify-tag -v "${1}"; then
+    echo "${1} is a valid signed tag"
+    return 0
+  else
+    echo "Git tag = ${1} signature is NOT valid. Codesigning will be skipped..."
+    return 1
+  fi
+}
+
+CURRENT_DIR="${PWD}"
+
+# Step 1
+echo "" && echo "=== Get latest prod travis build id and commit sha ===" && echo ""
+
+zen_tag="$(curl -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/HorizenOfficial/zen/git/refs/tags | jq -r '[.[] | select(.ref | test("refs/tags/v[0-9]\\.[0-9]\\.[0-9]$"))][-1].ref' | sed -e 's|refs/tags/||')"
+check_runs="$(curl -sL -H "Accept: application/vnd.github+json" -H "X-GitHub-Api-Version: 2022-11-28" "https://api.github.com/repos/HorizenOfficial/zen/commits/${zen_tag}/check-runs")"
+travis_build_id="$(basename "$(jq -rc '.check_runs[0].details_url' <<< "${check_runs}")")"
+commit_sha="$(jq -rc '.check_runs[0].head_sha' <<< "${check_runs}")"
+MAINTAINER_KEYS="219f55740bbf7a1ce368ba45fb7053ce4991b669 8EDE560493C65AC1 D3A22623FF9B9F11 1FCA7260796CB902 F136264D7F4A2BB5"
```

Comment:
Whose keys are those?

Reply:
Same as these ones
https://github.com/HorizenOfficial/zen_ci_testing/blob/2e934e18cb3992973a94c6686948bd9d353bf88b/contrib/ci-horizen/scripts/release/package_release.sh#L13

So #cronic, PaoloT, DanileR, AlessandroP, GiacomoP

Reply:
Shall we move this into TRAVIS ENV var better? Naming for example `ZEN_REPO_MAINTAINER_KEYS`
In case we need to change adjust we will not need to modify the code and submit PR